.gitignore

```diff
@@ -0,0 +1,2 @@
+/.bsp/
```

Comment:
Could you add some more things to ignore [like ](https://github.com/nationalarchives/dr2-ingest-parsed-court-document-event-handler/blob/main/.gitignore)?